cmd/root.go

```diff
@@ -3,6 +3,9 @@ package cmd
 import (
 	"context"
 	"fmt"
+	"log" //nolint:depguard // ignore depguard for this file
```

Comment:
As mentioned elsewhere, this is done to make sure linter doesn't yell at `log` and `slog` being imported in a non-main file. To get rid of this `//nolint`, we could add a `Fatal` log level in our logger instead. But to use it without using a default/global logger, it would mean creating a logger within the `Execute` and `startScanner` functions in this file.

LMKWYT.